.github/workflows/e2e_run_all.yaml

```diff
@@ -225,19 +225,28 @@ jobs:
       matrix:
         container_runtime:
           - crio
+        cluster_type:
+          - onprem
         os:
           - ubuntu
         provider:
           - generic
         arch:
           - amd64
+        include:
+          - container_runtime: containerd
+            cluster_type: eks
+            os: ubuntu
+            provider: generic
+            arch: amd64
```

Comment:
Could we refactor this into a table for more readability?

Reply:
Yes, we can!

Reply:
done. that's what you wanted @stevenhorsman ?